src/r2x/parser/plexos.py

```diff
@@ -144,11 +148,14 @@ def __init__(self, *args, xml_file: str | None = None, **kwargs) -> None:
         assert self.config.run_folder
         self.run_folder = Path(self.config.run_folder)
         self.system = System(name=self.config.name, auto_add_composed_components=True)
-        self.property_map = self.config.defaults["plexos_input_property_map"]
-        self.device_map = self.config.defaults["plexos_device_map"]
-        self.fuel_map = self.config.defaults["plexos_fuel_map"]
-        self.device_match_string = self.config.defaults["device_name_inference_map"]
-        self.generator_models = self.config.defaults["generator_models"]
+        self.property_map = self.config.defaults["plexos_input_property_map"] or {}
+        self.device_map = self.config.defaults["plexos_device_map"] or {}
+        self.fuel_map = self.config.defaults["plexos_fuel_map"] or {}
+        self.category_map = self.config.defaults["plexos_category_map"] or {}
+        self.device_match_string = self.config.defaults["device_name_inference_map"] or {}
+        self.generator_models = self.config.defaults["generator_models"] or {}
+        self.year = self.config.solve_year
+        assert isinstance(self.year, int)
```

Comment:
@pesap Is there a reason you added this here? It causes crash since we look for study year at line 181. The if statement `if not self.year` can never be true since we are asserting it is an instance of `int` here.